buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -171,8 +171,7 @@ class BuildPlugin implements Plugin<Project>  {
                     project.rootProject.ext.hadoopClient = ["org.apache.hadoop:hadoop-client:$version"]
                     println "Using Apache Hadoop on YARN [$version]"
                     break
-
-                default:
+                case "hadoopStable":
                     String version = project.hadoop22Version
                     project.rootProject.ext.hadoopVersion = version
                     project.rootProject.ext.hadoopClient = ["org.apache.hadoop:hadoop-client:$version"]
```

Comment:
There's no difference in the code of the two different switch paths other than the `println`, can be move the settings out? Unless we need to *not* set them in the 'default' case (in which case we should be explicit in that)